index.html

```diff
@@ -175,5 +175,36 @@ <h2 class="skill_h2">Skills</h2>
       </div>
     </div>
   </section>
+  <section class="contact-container">
+    <div class="h2_contact">
+      <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    </div>
+    <form action="https://formspree.io/f/mknagbor" method="POST" class="form_container">
+      <div class="input_form">
+        <input type="text" name="full_name" id="name" placeholder="Full name" maxlength="30" required>
+      </div class="input_form">
```

Comment:
- [OPTIONAL] I highly recommend that you remove this class because it placed in the incorrect place